src/task1/task1.ts

```diff
@@ -0,0 +1,26 @@
+import { pipeline, Transform } from 'stream';
+
+const readStream = process.stdin;
+const writeStream = process.stdout;
+
+class MyTransformStream extends Transform {
+  _transform(chank, encoding, callBack) {
+    try {
+      const inputString: string = chank.toString();
+      const resultString = [...inputString.split('').reverse(), '\n'].join('');
+      callBack(null, resultString);
+    } catch (error) {
+      callBack(error);
+    }
+  }
+}
+
+const myTransformStream = new MyTransformStream();
+
+pipeline(readStream, myTransformStream, writeStream, (err) => {
```

Comment:
it will be better to add separate error handler:
const handleError = ...
pipeline(readStream, myTransformStream, writeStream, handleError);